app/helpers/migration_helper.rb

```diff
@@ -0,0 +1,47 @@
+module MigrationHelper
+  def migration_started_at(data_migrations)
+    data_migrations.map(&:started_at).compact.min || Time.zone.now
```

Comment:
I was wondering why do we need to default to the current time? If the migration hasn't started wouldn't we want to always return nothing 🤔 ?

Reply:
Yeah good point, returning `nil` would make more sense here I'll update

Reply:
Ah actually I realised why I did this; when you prepare a migration it creates all the `DataMigration` instances in-band (so we know a migration has started) but there's a short delay before the worker picks up the job where all the `DataMigration` models are pending (so none have a `started_at`). In this scenario I figured the best thing would be to display as if the migration has only just started by returning `Time.zone.now`. I've added a comment to that helper for clarity.

Reply:
ah makes sense, thanks 👍